packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesAppLoader.m

```diff
@@ -136,7 +136,7 @@ - (void)startLoadingFromManifest:(EXUpdatesUpdate *)updateManifest
       NSLog(@"EXUpdatesAppLoader: Loaded an update with the same ID but a different scopeKey than one we already have on disk. This is a server error. Overwriting the scopeKey and loading the existing update.");
     }
 
-    if (existingUpdate && existingUpdate.status == EXUpdatesUpdateStatusReady) {
+    if (existingUpdate && (existingUpdate.status == EXUpdatesUpdateStatusReady || existingUpdate.status == EXUpdatesUpdateStatusFailed)) {
```

Comment:
I think this is resolved in a different way; based on last week's conversation, we now only mark updates that threw an error before the initial render as "failed" (in which case we don't run them again) and otherwise we will try them again.